src/pages/networks/actions/DeleteLocalPeerBtn.tsx

```diff
@@ -0,0 +1,103 @@
+import type { FC } from "react";
+import { useState } from "react";
+import { useQueryClient } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import {
+  ConfirmationButton,
+  Icon,
+  useNotify,
+  useToastNotification,
+} from "@canonical/react-components";
+import { useCurrentProject } from "context/useCurrentProject";
+import ResourceLabel from "components/ResourceLabel";
+import type { LxdNetwork } from "types/network";
+import { useNetworkEntitlements } from "util/entitlements/networks";
+import { deleteNetworkPeer } from "api/network-local-peering";
+import ResourceLink from "components/ResourceLink";
+
+interface Props {
+  network: LxdNetwork;
+  localPeering: string;
+}
+
+const DeleteLocalPeerBtn: FC<Props> = ({ network, localPeering }) => {
+  const notify = useNotify();
+  const [isLoading, setLoading] = useState(false);
+  const queryClient = useQueryClient();
+  const { canEditNetwork } = useNetworkEntitlements();
+  const { project } = useCurrentProject();
+  const projectName = project?.name || "";
+  const toastNotify = useToastNotification();
+  const networkURL = `/ui/project/${encodeURIComponent(project?.name ?? "")}/network/${encodeURIComponent(network.name)}`;
+
+  const onFinish = () => {
+    toastNotify.success(
+      <>
+        Local peering{" "}
+        <ResourceLabel type={"peering"} value={localPeering} bold /> deleted for
+        network{" "}
+        <ResourceLink type="network" value={network.name} to={networkURL} />.
+      </>,
+    );
+  };
+
+  const handleDelete = () => {
+    setLoading(true);
+    deleteNetworkPeer(network.name, projectName, localPeering)
+      .then(onFinish)
+      .catch((e) => {
+        notify.failure("Local peering deletion failed", e);
+      })
+      .finally(() => {
+        setLoading(false);
+        queryClient.invalidateQueries({
+          queryKey: [
+            queryKeys.projects,
+            projectName,
+            queryKeys.networks,
+            network.name,
+            queryKeys.peers,
+          ],
+        });
+      });
+  };
+
+  return (
+    <ConfirmationButton
+      loading={isLoading}
+      confirmationModalProps={{
+        title: "Confirm delete",
+        children: (
+          <p>
+            This will permanently delete local peering{" "}
+            <ResourceLink
+              type={"peering"}
+              value={localPeering}
+              to={`${networkURL}/local-peering`}
```

Comment:
```suggestion
              to={`${networkURL}/local-peerings`}
```
Using the correct url here